apps/admin-x-settings/test/acceptance/general/seometa.test.ts

```diff
@@ -12,8 +12,7 @@ test.describe('SEO Meta settings', async () => {
                     config: {
                         ...responseFixtures.config.config,
                         labs: {
-                            ...responseFixtures.config.config.labs,
-                            ui60: true
+                            ...responseFixtures.config.config.labs
                         }
                     }
                 }
```

Comment:
The whole `browseConfig` block can be removed now, as we're not overriding anything. Also note: there is a `toggleLabsFlag` utility you can use instead of having to override `browseConfig` to set a labs flag